### DISCUSSION TITLE
Why cannot use emojis in the articles?

### DISCUSSION THREAD

**Discussion by locksoft** [WordPress/gutenberg#66501] [Q&A]
Whenever I try to add one or more emojis in my WP posts I get an error when I try to save them:

"Failed update. Cannot insert the article in the database" (free translation from my language).

**Comment by ellatrix** [comment]
It should be possible to add emoji to your WordPress posts. You can try it here: https://playground.wordpress.net
Could it be an issue with the database or your host? Have you tried it on other sites?
